Index: src/object/OBJ_Lantern.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/object/OBJ_Lantern.java b/src/object/OBJ_Lantern.java
new file mode 100644
--- /dev/null	(date 1733508728061)
+++ b/src/object/OBJ_Lantern.java	(date 1733508728061)
@@ -0,0 +1,17 @@
+package object;
+
+import DannyGermanSimulator.GamePanel;
+import Entity.Entity;
+
+public class OBJ_Lantern extends Entity {
+    public  OBJ_Lantern(GamePanel gp){
+        super(gp);
+
+        type = type_light;
+        name = "Lantern";
+        down1 = setUp("/objects/lantern",gp.tileSize,gp.tileSize);
+        description = "[" + name + "]\nIlluminates your surroundings.";
+        price = 200;
+        lightRadius = 800;
+    }
+}
